pkg/controller/broker_client_manager.go

```diff
@@ -96,7 +97,7 @@ func (m *BrokerClientManager) RemoveBrokerClient(brokerKey BrokerKey) {
 	m.mu.Lock()
 	defer m.mu.Unlock()
 
-	glog.V(4).Info("Removing OSB client for broker %s", brokerKey.String())
+	glog.V(4).Infof("Removing OSB client for broker %s", brokerKey.String())
```

Comment:
This is a correct change. Format string wasn't being used as one.

@duglin do you want a `%q`?

Reply:
yes `%q` please

Reply:
yes `%q` please

Reply:
yes, please use a `%q`

Reply:
yes, please use a `%q`

Reply:
yes please use %q

Reply:
yes please use %q

Reply:
yes please use %q